components/Card/Card.tsx

```diff
@@ -1,34 +1,33 @@
-// TODO: #330 Refactor componente "Card"
-import { Button } from "components/Button/Button";
+import { Link } from "components/Link/Link";
 import { Logo } from "components/Logo/Logo";
 import { State } from "components/State/State";
 import Image from "next/image";
+import { Toy } from "types/common";
+import { urlFor } from "../../lib/sanity-client";
 
-export interface CardProps {
+export interface CardProps extends Toy {
   image?: { src: string; alt: string };
   title: string;
   available: boolean;
-  description: string;
   link: string;
 }
```

Comment:
Que te parece si renombramos las props para que se llamen como vienen definidas en el type `Toy`?

Reply:
> Que te parece si renombramos las props para que se llamen como vienen definidas en el type `Toy`?

Hola, ya agregué los cambios, sobre la interfaz CardProps no entiendo, si ya extiende de Toy para que le agregaría el mismo nombre? No es mejor en lugar de extender CardProps agregar el tipo Toy a las props del componente Card?

Tampoco entiendo que es cada cosa que viene del tipo Toy y que sustituye, si me pudieras dar una mano, sería de mucha ayuda

Reply:
Sisi @ErikGIovani, estuve viendo otra vez el type `Toy` y noté que me había confundido cuando escribí el issue, perdón.
Dejemos como estaba, sin extender a `Toy`...

```ts
interface CardProps {
  image: SanityImage; // <= Esto habria que agregarlo de "types/common"
  title: string;
  available: boolean;
  link: string;
  description: string // <= Esta prop ya la agrego Nykka, esta en la rama development
}
```

Y al renderizar la imagen podrías chequear si `image` no es `undefined`
También `urlFor` quedaría así: `urlFor(image).width(528).url()`

Reply:
> Sisi @ErikGIovani, estuve viendo otra vez el type `Toy` y noté que me había confundido cuando escribí el issue, perdón. Dejemos como estaba, sin extender a `Toy`...
> 
> ```ts
> interface CardProps {
>   image: SanityImage; // <= Esto habria que agregarlo de "types/common"
>   title: string;
>   available: boolean;
>   link: string;
>   description: string // <= Esta prop ya la agrego Nykka, esta en la rama development
> }
> ```
> 
> Y al renderizar la imagen podrías chequear si `image` no es `undefined` También `urlFor` quedaría así: `urlFor(image).width(528).url()`
> 
> Creo que con eso estaría!

Gracias @TonyMckes , ya hice los cambios.